srcpkgs/zsh-autosuggestions/template

```diff
@@ -0,0 +1,19 @@
+# Template file for 'zsh-autosuggestions'
+pkgname=zsh-autosuggestions
+version=0.4.3
+revision=1
+noarch=yes
+depends="zsh"
+short_desc="Fish-like autosuggestions for zsh"
+maintainer="vinnyA3 <vincent.aceto@gmail.com>"
+license="MIT"
+homepage="https://github.com/zsh-users/zsh-autosuggestions"
+distfiles="${homepage}/archive/v${version}.tar.gz>${pkgname}-${version}.tar.gz"
```

Comment:
There is no need to rename the distfiles

Reply:
Fixed distfiles rename